src/app/space/create/deployments/apps/deployment-details.component.html

```diff
@@ -26,7 +26,7 @@
           <pfng-chart-sparkline class="chart-sparkline" [config]="memConfig" [chartData]="memData"></pfng-chart-sparkline>
         </div>
         <div class="chart-column-right">
-          <deployment-graph-label [ngClass]="memLabelClass" type="Memory" [dataMeasure]="memUnits" [value]="memVal" [valueUpperBound]="memMax"></deployment-graph-label>
+          <deployment-graph-label [dataMeasure]="memUnits" type="Memory" [ngClass]="memLabelClass" [value]="memVal" [valueUpperBound]="memMax"></deployment-graph-label>
```

Comment:
Same question here as I had for the other file - is there some style guide reasoning for changing the order of the attributes?